cmake/README.md

```diff
@@ -35,8 +35,10 @@ cd /tmp/my-build
 
 ### Configure
 
+In the build directory, specify the oneTBB source directory to generate a build system for it.
```

Comment:
I don't personally think that this comment adds useful information here. Flow of this README already implies that the user's current working directory to be the build directory after the steps from "Preparation" section.

Reply:
agree

Reply:
It might be the case for users who are familiar with `cmake`, but for users who don't, it can be very confusing because they have no idea why running an unfamiliar command in an empty directory and what does it do.

Reply:
But if you guys strongly against it, feel free to suggest a change and I will follow.

Reply:
https://github.com/oneapi-src/oneTBB/blob/master/cmake/README.md#configure-build-and-test is a step-by-step guide to configure, build and test oneTBB. So the document is assuming that the reader follows it from top to bottom.
I think you could probably replace <repo_root> by <onetbb_source_directory> and add `From the build directory, run:` as it has been done for INSTALL.md. Will it help?

Reply:
And yes, it's fine to assume the developer is reading from top to bottom; however, in my opinion, it's also very import to descript what each step does in a README doc. Especially for `cmake` which has a very different build workflow than other common programming languages (e.g., Java, Python, Go).